_includes/events/advtable-events.md

```diff
@@ -0,0 +1,7 @@
+The following events are provided by the [Advanced Tables plugin]({{ site.baseurl }}/plugins/premium/advtable/).
+
+| Name | Data | Description |
+|------| ---- | ----------- |
+| newrow | N/A | Fired when a row is created |
+| newcell | N/A | Fired when a cell is created |
+| TableModified | `{ table: HTMLTableElement }` | Fired when style or structural changes are made to a table. <br/>> **Note**: This feature is only available for Advanced Tables 1.0.2 and later. |
```

Comment:
Can we share docs between this TableModified event and the table plugin TableModified event?

Reply:
maybe, it's tricky because it's a markdown table... so it would be fiddley, hacky, and could break at any time.